src/chess/Chess.java

```diff
@@ -1,5 +1,5 @@
 
-package Chess;
+package chess;
```

Comment:
File names in java are usually the name of the class I'm pretty sure, so by convention it should probably be `chess` for the package name and `Chess.java` for the file name (assuming the name of the class is `Chess`)